src/server/services/ProductHandler.test.js

```diff
@@ -0,0 +1,49 @@
+const expect = require('unexpected');
+
+// TODO: ProductHandler should start with lowercase
+const productHandler = require('./ProductHandler');
+const Product = require('../../model/Product');
+
+describe('Product Handler', () => {
+  const sut = productHandler();
+  let product1;
+  let product2;
+
+  context('given no products', () => {
+    beforeEach(() => {
+      sut.setProducts([]);
+      product1 = new Product('1', ['Bread'], ['kg'], true, 3, 'Food');
```

Comment:
to avoid repeating this code in this test, you can create a helper function something like: `createProduct(id)`